### DISCUSSION TITLE
Doubt when executing the migration from 16->17

### DISCUSSION THREAD

**Discussion by Tobiewe** [OCA/OpenUpgrade#4540] [Q&A]
I have been reading the documentation but when I execute the command, from the env of Odoo it tells me the following. 
    - odoo-bin server: error: option --upgrade-path: the path '/home/user/dev/checkouts/project/no_repo/odoo/src/  OpenUpgrade/openupgrade_scripts/scripts' is not a valid upgrade directory.

If someone can answer my question it would be great. 
I would also like to know if when migrating from 16->17 it is necessary to have OCA 17 or OCA 16.
Thank you

**Comment by legalsylvain** [comment]
Hi. openupgrade 17 is not yet available So it can not work for the time being.

kind regards.
